bundles/org.openhab.binding.openthermgateway/src/main/java/org/openhab/binding/openthermgateway/internal/DataItemGroup.java

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.openthermgateway.internal;
+
+import java.util.HashMap;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.smarthome.core.library.unit.SIUnits;
+
+/**
+ * @author Arjen Korevaar - Initial contribution
+ */
+@NonNullByDefault
+public class DataItemGroup {
+
+    public static final HashMap<Integer, DataItem[]> dataItemGroups = createDataItemGroups();
```

Comment:
Please use the interface instead of the actual implementation for declarations and return values. 
```suggestion
    public static final Map<Integer, DataItem[]> dataItemGroups = createDataItemGroups();
```

Reply:
Adjusted